.github/workflows/go.yml

```
@@ -24,14 +25,16 @@ jobs:
       with:
         go-version: ${{ matrix.version }}
 
-    - name: Build Linux
-      run: GOOS=linux go build -v .
+    - name: Build
+      run: go build -v .
```

@AmeyaPurao  try fixing it this way?

```suggestion
    - name: Build Linux
      run: go build -v .
      env:
        GOOS: linux
```

---

i can but the windows ci is just slow and we only need to compile it on linux day of so is there really a need to try to compile the linux version from the windows host? I only compile the windows version on windows for the tests rn

---

Sounds good 👍